extensions/blocks/anchor-fm/editor.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+import { dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { name } from '.';
+import getJetpackExtensionAvailability from '../../shared/get-jetpack-extension-availability';
+import { waitForEditor } from '../../shared/wait-for-editor';
+
+async function insertSpotifyBadge() {
+	const { spotifyShowUrl } = window.Jetpack_AnchorFm;
+	if ( ! spotifyShowUrl ) {
+		return;
+	}
+
+	await waitForEditor();
+
+	dispatch( 'core/block-editor' ).insertBlock(
+		createBlock( 'core/image', {
+			url: 'https://cldup.com/Dv6JZWyRpq-1200x1200.png',
```

Comment:
The previous discussion (https://github.com/Automattic/jetpack/pull/17936#discussion_r533451558) has been lost after the last commit, so copying it here again for visibility:

> Where can we host this officially? I think it should be loaded from some global CDN address and not from the specific Jetpack directory of the current site, since these links will break after the plugin gets deactivated. They should work on their own, since they are a simple image block.
> 
> I haven't found a good place where Spotify hosts these and can be hotlinked. I've only been able to download this badge as a ZIP file from the dashboard.

> @sfougnier Do you have access to the "official" spotify badge? Is this something we'd want to have a SVG?

Reply:
reposting my reply too :D 

Here is the official archive I got from the dashboard https://podcasters.spotify.com/resources/spotify-podcast-badges.zip. It includes SVGs too. We just probably need to pick one/more and host it somewhere. Maybe widgets.wp.com if there is nothing more suitable?

Reply:
That's a good question.

> Maybe widgets.wp.com if there is nothing more suitable?

I would have voted for WordPress.com as well, it seems like the best course of action.

Since the block is available to folks that may not have connected their site to WordPress.com, I was a bit worried we couldn't hotlink from WordPress.com yet. but it does make for a better user experience if folks deactivate Jetpack later on, and we do have a precedent with the fallback image for Open Graph Meta tags:
https://github.com/Automattic/jetpack/blob/3dfc97c0d9e26dcfb96899e0175f85c56b827a48/functions.opengraph.php#L396

Reply:
I think this should be something like `Jetpack::staticize_subdomain( 'https://wordpress.com/i/spotify-badge.svg' )`. It [looks like](https://github.com/Automattic/jetpack/blob/07a725146df465cc8df0f08adaa840cd41a4f582/class.jetpack.php#L5807) that'll give us a properly randomized static subdomain (as opposed to hard-coding it). Or – wait, `s-ssl`, which I've never seen before. But that `i` directory on dotcom definitely looks like the place to put it. 🙃

Reply:
Added the image to WP.com in D53943-code